### PR TITLE
feature/add github extensions

### PR DESCRIPTION
**Pull request by dmitry-mightydevops**
- feat: add github extensions

See video with explanation: https://drive.google.com/file/d/1abda1OzvHFNw2ZZWAfQeLlmd_v0O1_kk/view?usp=sharing

### CODE REVIEW

**Comment by kseniyashaydurova** [comment]
@dmitry-mightydevops It seems that video has no sound :(

File: github/extensions.sh [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    fi
+
+    [[ ${#labels[@]} -eq 0 ]] && labels=("enhancement")
+    [[ ${#assignees[@]} -eq 0 ]] && assignees=("${OPS[@]}") 
+
+    if [[ dryrun -eq 1 ]]; then
+        command="gh pr create -B $branch -p $project `echo ${labels[@]/#/-l }` `echo ${assignees[@]/#/-a }` `echo ${OPS[@]/#/-r }` --fill"
+        echo $command
+    else
+        gh pr create -B $branch -p $project `echo ${labels[@]/#/-l }` `echo ${assignees[@]/#/-a }` `echo ${OPS[@]/#/-r }` --fill
```
Comment (kseniyashaydurova) [inline github/extensions.sh:92]: Is it possible to define `command` before if/else block to DRY, so the command would be executed from `command` variable?

**Review by kseniyashaydurova — APPROVED** [review]

File: github/extensions.sh (hunk 1 above)
Comment (dmitry-mightydevops) [reply github/extensions.sh:92]: so this would require the use of "eval" which is not safe :)

**Comment by dmitry-mightydevops** [comment]
> @dmitry-mightydevops It seems that video has no sound :(
Well the video is self explanatory, you just need to put in your repo that `export OPS=...` so that you can automate the creation of the github pr. 

I.e. this CLI would look into that env var and immediately assigns the reviewers as defined. 

Much easier than to type the usernames each time or do that through the GUI clicks.

We can talk more at the devops standup.

**Review by populov — APPROVED** [review]

**Comment by roman-mightydevops** [comment]
Approved

**Comment by roman-mightydevops** [comment]
Approved

**Pull request merged**